### DISCUSSION TITLE
Percentage value of discount not displayed in checkout object.

### DISCUSSION THREAD

**Discussion by DidoMarchet** [saleor/saleor#10528] [General] [answered]
Hi,
if we add a voucher with a Percentage value
its value cannot be displayed.

For example we have a voucher 25%.
If I retrieve the checkout its amount is 0 and there's no way to get its value.
It could be confusional for customers because they see the price changing but with discount amount 0 and no reference to the percentage.

```json
{
   "data":{
      "checkoutAddPromoCode":{
         "checkout":{
		 	// ORDER ATTRIBUTES
			
            "isShippingRequired":true,
            "discount":{
               "currency":"USD",
               "amount":0.0
            }
            "discountName":null,
            "translatedDiscountName":"",
            "voucherCode":"WELCOME25",
			
		 	// ORDER ATTRIBUTES
         },
         "errors":[
            
         ]
      }
   }
}
```

Any suggestion?

Thanks in advance and kind regards,

Davide

**Comment by karolkielecki** [comment]
Hi @DidoMarchet, I'll investigate it with the team and get back to you soon.

**Reply by DidoMarchet** [reply]
Hi @karolkielecki , 
do you have any update?
It's not a blocking issue but it creates misunderstanding.

Really thanks and kind regards,

Davide

**Comment by DidoMarchet** [comment]
@karolkielecki do you have any update about this issue?

We also saw this issue recurs  in *dashboard v3.1.0-dev core v3.8.0-a* if you use a voucher to some product and not on the entire orders.

Can you have a check because because some customers have reported.

Thanks in advance and kind regards,  Davide

**Comment by CossackDex** [comment]
Hi @DidoMarchet,

I'm transferring this discussion to an issue, could you please share the exact details of a voucher that you created?

**Comment by DidoMarchet** [comment]
Hi @CossackDex ,
do you have any update related of this issue? 
I know about priorities and I'm sorry to ask but I think this issue it's a little bit annoying because prices and payment are unclear. 

Related to: [https://github.com/saleor/saleor/issues/10782](https://github.com/saleor/saleor/issues/10782)


Thanks and kind regards,

Davide

**Reply by karolkielecki** [reply, marked as answer]
Hi @DidoMarchet, I've just raised this issue to our team again -- It's a known bug. In such cases, there is no option to easily check the voucher discount amount. We're currently fully booked with v3.9, so there is no ETA for the fix. We'll keep you posted, and thanks for understanding.
